.eslintrc.js

```diff
@@ -2,4 +2,18 @@ module.exports = {
   extends: ['@alhadis/eslint-config'],
```

Comment:
**Tip:** ESLint allows you to shorten the name of [shareable configs](https://eslint.org/docs/latest/developer-guide/shareable-configs#using-a-shareable-config) if they match one of these formats:

```
"@user/eslint-config" -> "@user"
"eslint-config-user"  -> "user"
```

```suggestion
  extends: '@alhadis',
```